Spanner/tests/System/PgQueryTest.php

```diff
@@ -367,9 +367,14 @@ public function testBindTimestampParameter()
                 'p1' => self::$timestampVal
             ]
         ]);
-
         $this->assertCount(2, iterator_to_array($res));
 
+        $res = self::$database->execute("SELECT * FROM " . self::TABLE_NAME . " WHERE dt BETWEEN $1 AND $2", [
+            'parameters' => [
+                'p1' => new Date(new \DateTime('2020-01-01')),
+                'p2' => new Date(new \DateTime('2021-01-01'))
+            ]
+        ]);
```

Comment:
You can remove this.. 
Move the assertCount to the last (maybe a call to `reset` is needed after using `current`)